example_server.ini

```diff
@@ -3,19 +3,20 @@
 # won't parse them correctly!  Also note that the decoded keys are in OpenSSL
 # byte order, which is opposite from the byte order stored in the pnNetBase
 # .hpp files in Cyan's original source.
+# See Docs/server.ini.md for more details.
 
 # The following keys and addresses represent the official MOULagain server
-Server.Game.N "kEso0EnSJNkgyQtVyUO8IGAzy9cVgtEZG3Dy/s0urdYgu+omUP0/3sQihN5EOJqqXQl2ahI7cifHptdDafekOA=="
-Server.Game.X "Pds9j7NYWE9o6WV6o88EhGczzxR/NpNimHn9yLq2X9xoAdiEH1x2R1w/O5nwWv3CPmVEPgm9IalIcr0CvZOy+Q=="
+Server.Game.N "kEso0EnSJNkgyQtVyUO8IGAzy9cVgtEZG3Dy/s0urdYgu+omUP0/3sQihN5EOJqqXQl2ahI7cifHptdDafekOA=="
+Server.Game.X "Pds9j7NYWE9o6WV6o88EhGczzxR/NpNimHn9yLq2X9xoAdiEH1x2R1w/O5nwWv3CPmVEPgm9IalIcr0CvZOy+Q=="

```

Comment:
What's with the (per GitHub) no-op changes? I'm assuming you changed the line endings from CRLF to LF, but I just want to make sure.

Reply:
I was wondering what was up with that - IntelliJ also complained about the line endings in a way that didn't quite make sense... I checked again, and apparently the file used *mixed* CRLF and LF line separators - ugh! Fixed now in a separate commit, so the individual commit diffs should be clean.